api/handlers/searchhandler.py

```diff
@@ -0,0 +1,31 @@
+import datetime
+import elasticsearch
+from requests import ConnectionError
+
+from .. import base
+from .. import config
+
+log = config.log
+
+es_client = elasticsearch.Elasticsearch([config.get_item('elasticsearch', 'es_endpoint')])
+
+
+class SearchHandler(base.RequestHandler):
+
+    def __init__(self, request=None, response=None):
+        super(SearchHandler, self).__init__(request, response)
+
+    def get(self, cont_name=None, **kwargs):
+        if self.public_request:
+            self.abort(403, 'search is available only for authenticated users')
+        size = self.get_param('size')
+        body = self.request.json_body if self.request.body else {
+          'query': {
+            'match_all': {}
+          }
```

Comment:
Am I correct in thinking this... defaults to returning all documents if no query was passed?